apps/server/src/shared/repo/news/news.repo.integration.spec.ts

```diff
@@ -102,7 +102,7 @@ describe('NewsRepo', () => {
 			};
 			const pagination = { skip: 0, limit: 20 };
 
-			const [result, count] = await repo.findAll([target], false, { pagination });
+			const [result, count] = await repo.findAllPublished([target], { pagination });
```

Comment:
Here is missing tests for findAllUnpublished

Reply:
Added tests